src/common/data/enums/TransactionKind.ts

```diff
@@ -1,6 +1,7 @@
 enum TransactionKind {
-  SEND = 'Send',
+  SEND = 'Sent',
   RECEIVE = 'Received',
+  SELF = 'SELF',
```

Comment:
Also, we don't need a Self type, there would be no transaction with a transactionType variable set to Self.

Reply:
We can do it later as well